Spigot-Server-Patches/0010-Adventure.patch

```diff
@@ -2324,6 +2332,25 @@ index b51a874e4665f977a154792e6216e03e04525f39..6ab14bccb1fcd108931bf7ec331e60f6
 +    }
 +
 +    @Override
++    @SuppressWarnings("unchecked") // safe casts
++    public <T> java.util.Optional<T> get(final net.kyori.adventure.pointer.Pointer<T> pointer) {
++        if (pointer == net.kyori.adventure.identity.Identity.NAME) {
++            return java.util.Optional.of((T) this.getProfile().getName());
++        } else if (pointer == net.kyori.adventure.identity.Identity.DISPLAY_NAME) {
++            return java.util.Optional.of((T) this.displayName());
++        } else if (pointer == net.kyori.adventure.identity.Identity.UUID) {
++            return java.util.Optional.of((T) this.getUniqueId());
++        } else if (pointer == net.kyori.adventure.permission.PermissionChecker.POINTER) {
++            if (this.adventure$permissions == null) {
++                // todo: allow properly returning NOT_SET?
```

Comment:
Any suggestions on how to handle this?

Reply:
There's a superperms method for `isPermissionSet`, you could use that?